src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -35,13 +36,14 @@ public class DefaultHtmlGenerator : IHtmlGenerator
         /// </summary>
         public DefaultHtmlGenerator(
             [NotNull] AntiForgery antiForgery,
-            [NotNull] IScopedInstance<ActionBindingContext> bindingContextAccessor,
+            [NotNull] IOptions<MvcOptions> optionsAccessor,
             [NotNull] IModelMetadataProvider metadataProvider,
             [NotNull] IUrlHelper urlHelper,
             [NotNull] IHtmlEncoder htmlEncoder)
         {
             _antiForgery = antiForgery;
-            _bindingContextAccessor = bindingContextAccessor;
+            var clientValidatorProviders = optionsAccessor.Options.ClientModelValidatorProviders.ToArray();
```

Comment:
nit: why is this here (versus in the `CompositeClientModelValidatorProvider` constructor)?